src/v3/src/components/List/ListItemText.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Box } from '@okta/odyssey-react-mui';
+import { FunctionComponent, h } from 'preact';
+
+import { useHtmlContentParser } from '../../hooks';
+
+type ListItemTextProps = {
+  text: string;
+};
+
+const ListItemText: FunctionComponent<ListItemTextProps> = ({ text }) => {
```

Comment:
What is this new component for?

Reply:
This is so we can use html in list item text.  Im using it [here](https://github.com/okta/okta-signin-widget/pull/3632/files#diff-b34c14683e246f719bcf66a192426e681520d5b5678062427aee0be52535339cR400) to have bold text in a list.

Reply:
One last somewhat nit/comment: any idea why the gen3 spacing is so tight in the "Set up Okta Verify" view? (The view about 5 seconds into your demo vid).

This feels much tighter than normal but maybe that's how all of our lists are?

Reply:
actually that part isn't a list it just uses regular description elements.  But it does look tight not sure why, it looked like that even before my change.  Ill see if I can find why the description elements dont have any bottom margin

Reply:
I added an optional bottomMargin property to the description element to create some space